### ISSUE TITLE
Compressed kerning with trailing backtick

### ISSUE THREAD

**Issue by omenos** [intel/intel-one-mono#3]
It would seem that text wrapped in backticks will have compressed kerning when the character is an alphabetic symbol. For example, this is how the following text is rendered:

```
This is an example of `alphabetic trailing backtick`, while this is a `non-alpha backtick!`.
```

![image](https://github.com/intel/intel-one-mono/assets/14795448/3840b5d6-d634-40b7-aa2c-0d99ac7cf343)

**Comment by riklus** [comment]
I have this same issue too.
<img width="503" alt="Schermata 2023-05-13 alle 6 33 21 PM" src="https://github.com/intel/intel-one-mono/assets/82454869/f4202535-471c-463f-bd4f-35f6e462a431">

**Comment by Cecile-Lebleu** [comment]
Same here. I noticed it doesn't happen with all themes, for example, not with the Noctis collection of themes. 
![Screenshot 2023-05-13 at 1 17 22 PM](https://github.com/intel/intel-one-mono/assets/22694361/7f0b9314-321c-420c-9052-43cb660392ad)
![Screenshot 2023-05-13 at 1 17 09 PM](https://github.com/intel/intel-one-mono/assets/22694361/7bb5a7f5-b773-4784-baaf-9b9bcc6c5192)

**Comment by arjpar** [comment]
I also have the same issue.

**Comment by moyogo** [comment]
The dead marks feature in the feature code should be removed. This is a bad practice and makes the font unusable for programming languages, markup languages and language orthographiés that use those ASCII marks.

**Comment by slice** [comment]
I found a workaround.

In short, you have to disable the `ccmp` OpenType feature. You can do this in CSS by using e.g. `font-feature-settings: "ccmp" 0;`:

![image](https://github.com/intel/intel-one-mono/assets/4206232/e184dd33-ea3b-401a-aa3e-83318dc7e2dc)

Compare this to how it looks by default (incorrect):

![image](https://github.com/intel/intel-one-mono/assets/4206232/757bfe3a-c62b-483e-8a93-cce149829618)

This also works fine in other software that lets you customize which OpenType features are active. One such example is [kitty](https://sw.kovidgoyal.net/kitty/), a terminal emulator:

![image](https://github.com/intel/intel-one-mono/assets/4206232/d4df3a4c-4fd0-4c38-b67c-729fa16eb522)

Notice how the comment renders correctly.

I stumbled across this by using [the TTX tool from `fonttools`](https://fonttools.readthedocs.io/en/latest/ttx.html) to crudely rip through the font files, which lets me poke around in some XML (representational of the original font) to find out what could be causing this. I'm no type designer, but there appears to be some kind of glyph substitution machinery that is supposed to be used with other language systems, but appears to be (inadvertently?) enabled by default in Latin. I don't understand this completely, so take this explanation with a grain of salt. Here is some related TTX:

```xml
<Lookup index="18">
  <LookupType value="1"/>
  <LookupFlag value="0"/>
  <!-- SubTableCount=1 -->
  <SingleSubst index="0">
    <Substitution in="acute" out="uni0301"/>
    <Substitution in="breve" out="uni0306"/>
    <Substitution in="caron" out="uni030C"/>
    <Substitution in="cedilla" out="uni0327"/>
    <Substitution in="circumflex" out="uni0302"/>
    <Substitution in="dieresis" out="uni0308"/>
    <Substitution in="dotaccent" out="uni0307"/>
    <Substitution in="grave" out="uni0300"/>
    <Substitution in="hungarumlaut" out="uni030B"/>
    <Substitution in="macron" out="uni0304"/>
    <Substitution in="ogonek" out="uni0328"/>
    <Substitution in="ring" out="uni030A"/>
    <Substitution in="tilde" out="uni0303"/>
    <Substitution in="uni02CD" out="uni0331"/>
    <Substitution in="uniA788" out="uni032D"/>
  </SingleSubst>
</Lookup>
```

As evident in the sample above, this behavior occurs not only for grave (`), but also for others such as dieresis (¨); it gets substituted with its combining form, merging it with the previous character (in certain scenarios).

Is this supposed to simplify the insertion of accented characters in languages such as Catalan? I'm not sure. I'm also unsure as to what the intended fix is; I don't know much about professional font authoring and what the correct semantics are here, so I'll leave the proper fix to the designers.

**Comment by slice** [comment]
Addendum: the README says this (with added formatting, because the linebreaks don't render correctly on GitHub):

> **Language Support**: `ccmp`, `mark` and `locl` features ensure correct display across a wide range of languages. These are usually activated by default. We recommend setting the language tag/setting in your software to the desired language for best results.

So this appears to be an intentional feature of the font, albeit misfiring. Surrounding a word with grave characters is fairly prevalent in Markdown, code, and technical prose in general, and it's incorrect to substitute the glyph with a combining one in these cases. Probably a bug.

**Comment by elliotwesoff** [comment]
+1. The trailing backtick doesn't show up at all for me in neovim v0.9

**Comment by ipuustin** [comment]
Adding this line to `settings.json` in Visual Studio Code seems to work around the issue (at least for me):

    "editor.fontLigatures": "'ccmp' off",

**Comment by arjpar** [comment]
Should be fixed with the latest release (1.2.0)

**Comment by omenos** [comment]
Verifying this is fixed in 1.2.0, thanks for the update!
